web-src/src/pages/PageComposers.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <tabs-music></tabs-music>
+
+    <content-with-heading>
+      <template slot="heading-left">
```

Comment:
When you are using the grouped by name view, then you should also include the index list:

```vue
      <template slot="options">
        <index-button-list :index="composers_list.indexList"></index-button-list>
      </template>
```

Take a look at `PageAudiobooksArtists.vue`.